bridge-history-api/internal/config/config.go

```diff
@@ -30,6 +30,8 @@ type FetcherConfig struct {
 	GatewayRouterAddr        string `json:"GatewayRouterAddr"`
 	MessageQueueAddr         string `json:"MessageQueueAddr"`
 	BatchBridgeGatewayAddr   string `json:"BatchBridgeGatewayAddr"`
+	GasTokenGatewayAddr      string `json:"GasTokenGatewayAddr"`
+	WrappedTokenGatewayAddr  string `json:"WrappedTokenGatewayAddr"`
```

Comment:
what's the difference between `GasTokenGatewayAddr` and `WrppedTokenGateway`?

Reply:
GasTokenGateway is for user to deposit erc20 gas token.
WrppedTokenGateway is for user to deposit ether, and it will wrap ether to weth, and bridge weth as a normal erc20 token to layer 2.